pkg/operations/kubernetesupgrade/upgradecluster.go

```diff
@@ -70,7 +70,7 @@ func (uc *UpgradeCluster) UpgradeCluster(subscriptionID uuid.UUID, resourceGroup
 	}
 
 	switch ucs.OrchestratorProfile.OrchestratorVersion {
```

Comment:
It's not irrelevant to upgrade. Upgrade uses the version specified in its API model to set the container service API model to generate the template.
Following lines need to be changed and test:
1) https://github.com/Azure/acs-engine/blob/master/pkg/operations/kubernetesupgrade/v162upgrader.go#L42

2) https://github.com/Azure/acs-engine/blob/master/pkg/operations/kubernetesupgrade/v162upgrader.go#L27-L30

3) https://github.com/Azure/acs-engine/blob/master/pkg/operations/kubernetesupgrade/upgradecluster.go#L73-79